### ISSUE TITLE
two invalid urls in the repos when using newest oba branch 2.0

### ISSUE THREAD

**Issue by wkulesza** [OneBusAway/onebusaway-application-modules#339]
Using the newest 2.0 branch of the codebase and trying to build, there are two url's that are not valid anymore.
Can this be updated please ?
https://repo.maven.apache.org/maven2/org/onebusaway/onebusaway-siri-api-v20/1.0.3/onebusaway-siri-api-v20-1.0.3.jar
http://repo.obaweb.org:8080/archiva/repository/releases/

**Comment by sheldonabrown** [comment]
Are you on the main branch of onebusaway-application-modules?  I can't find any references to the retired maven repositories you reference.

**Comment by wkulesza** [comment]
We took the main repo and forked it, to work on correcting encoding as API responses are not in utf-8.
https://github.com/goeuropa/onebusaway-application-modules/tree/polishencoding is our work and the master branch says: "This branch is up to date with OneBusAway/onebusaway-application-modules:master. "

**Comment by wkulesza** [comment]
To be precise we took it from 2.0 here https://github.com/OneBusAway/onebusaway-application-modules/commit/2a254845fa4c0192908f6dc770a7cd23fc9fbe66

**Comment by wkulesza** [comment]
Maybe we should be using a different branch? We want to add fixes to encoding and change it to use utf-8

**Comment by sheldonabrown** [comment]
Yes, unified on https://github.com/camsys/onebusaway-application-modules/ is the very latest work.  I merge that back to the OneBusAway repo on occasion.  a PR against unified is your best bet.
